sample-plugin/pom.xml

```diff
@@ -45,6 +45,10 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>authorize-project</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>jobcacher</artifactId>
```

Comment:
FYI this should have been in test scope unless it is needed to compile the sample plugin. I'll fix this in one of my next few PRs, so this comment is just for future reference.